### ISSUE TITLE
zsh-autosuggestions highlighting has error

### ISSUE THREAD

**Issue by Songggggr** [jeffreytse/zsh-vi-mode#258]
If you delete with ctrl + u, it will reproduce
![image](https://github.com/jeffreytse/zsh-vi-mode/assets/131334123/f1778c7f-c485-40f5-9d46-93c3b0f3fa22)
![image](https://github.com/jeffreytse/zsh-vi-mode/assets/131334123/96d10646-0b36-4c0b-ac40-6f70370e9ef5)

**Comment by tummetott** [comment]
same here. took me a while to find out that this is cause by this plugin. please fix

**Comment by tummetott** [comment]
A workaround is to disable lazy loading by defining this variable before sourcing the pluing:

```
ZVM_INIT_MODE=sourcing
```

**Comment by Songggggr** [comment]
@tummetott It doesn't work, It can be repeated every time

**Comment by jeffreytse** [comment]
@Songggggr I'd love to dig this issue. Could you please provide much more details as issue template? Thank you.

**Comment by tummetott** [comment]
> @tummetott It doesn't work, It can be repeated every time

I mean the bug only exists when opening a new shell, typing something that has an autocompletion, canceling the line with C-u. Once you press ENTER for any cmd (can also be an empty cmd), then the bug is gone.

**Comment by Songggggr** [comment]
> @Songggggr I'd love to dig this issue. Could you please provide much more details as issue template? Thank you.



> @Songggggr I'd love to dig this issue. Could you please provide much more details as issue template? Thank you.



> > @tummetott It doesn't work, It can be repeated every time
> 
> I mean the bug only exists when opening a new shell, typing something that has an autocompletion, canceling the line with C-u. Once you press ENTER for any cmd (can also be an empty cmd), then the bug is gone.

Yes, press ENTER or reinput then the bug is gone. But repeated trigger autocomplete and Ctrl+u appearing again. @jeffreytse
